gpu/segmentation/include/pcl/gpu/segmentation/impl/gpu_extract_clusters.hpp

```diff
@@ -41,6 +41,25 @@
 
 #include <pcl/gpu/segmentation/gpu_extract_clusters.h>
 
+#include <cuda_runtime_api.h>
+#include <cuda.h>
+std::vector<int> economical_download(const pcl::gpu::NeighborIndices& indices,
+        std::vector<int>& sizes, int max_answers) {
```

Comment:
const reference?

Reply:
We do modify `sizes`, by downloading data into it `indices.sizes.download(sizes);`, We could download the data outside of the function, and then pass a const reference. Would you think that is a good idea?

Reply:
I don't think moving the download outside the function will affect either the name of the intended usage, so either way (const ref + download outside, or mut ref + comment for the reason) is fine by me

Reply:
Sounds great to me - thank you!